apps/web/components/Kbar.tsx

```diff
@@ -0,0 +1,234 @@
+import { SearchIcon } from "@heroicons/react/solid";
+import {
+  KBarProvider,
+  KBarPortal,
+  KBarPositioner,
+  KBarAnimator,
+  KBarSearch,
+  KBarResults,
+  useMatches,
+  useKBar,
+} from "kbar";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { isMac } from "@calcom/lib/isMac";
+import { Tooltip } from "@calcom/ui";
+
+// grab link to events
+// quick nested actions would be extremely useful
+const actions = [
+  {
+    id: "toggle-idle",
+    name: "Test Function",
+    section: "Status",
+    shortcut: ["t", "f"],
+    keywords: "set yourself away bookings",
+    perform: () => alert("Hello World"),
+  },
+  {
+    id: "upcoming-bookings",
+    name: "Upcoming Bookings",
+    section: "Booking",
+    shortcut: ["u", "b"],
+    keywords: "upcoming bookings",
+    perform: () => (window.location.pathname = "bookings/upcoming"),
```

Comment:
this should be router.push https://nextjs.org/docs/api-reference/next/router#userouter